naming/src/main/java/com/alibaba/nacos/naming/push/v2/task/PushExecuteTask.java

```diff
@@ -55,14 +55,14 @@ public PushExecuteTask(Service service, PushDelayTaskExecuteEngine delayTaskEngi
     @Override
     public void run() {
         try {
-            PushDataWrapper wrapper = generatePushData();
             for (String each : getTargetClientIds()) {
                 Client client = delayTaskEngine.getClientManager().getClient(each);
                 if (null == client) {
                     // means this client has disconnect
                     continue;
                 }
                 Subscriber subscriber = delayTaskEngine.getClientManager().getClient(each).getSubscriber(service);
+                PushDataWrapper wrapper = generatePushData(subscriber);
```

Comment:
If move generate into for-each, it will make the wrapper take no effect.
Because the push task will can't re-use the wrapper and do more duplicate calculation.

Reply:
Due to the ServiceInfo' instances info in PushDataWrapper is different to the subscriber, so I changed it to create by different  clientIP.

Reply:
You can do this specified calculation in push executor, the common task should only run common logic.